packages/deployed-backend-client/src/deployed_backend_client.test.ts

```diff
@@ -258,6 +258,7 @@ void describe('Deployed Backend Client', () => {
         additionalAuthTypes: [],
         conflictResolutionMode: undefined,
         apiId: 'awsAppsyncApiId',
+        modelSchemaS3Uri: 's3://bucketName/model-schema.graphql',
```

Comment:
Can we make this just the bucket name or ARN? We actually need a JSON file from that bucket that doesn't exist yet.

Reply:
Synced offline. We'll expose URI to requested file.